### ISSUE TITLE
Testing_Resposive_Query

### ISSUE THREAD

**Issue by janesteffi** [janesteffi/Testing_Resposive_Query#1]
1. Create a JS query to one tab to open at a time and others hide.
2. Make fields required when clicked on search button. 
3. Make page as responsive and make sure that it works on IE.
![Accordion](https://user-images.githubusercontent.com/72570229/96374441-8177b180-1190-11eb-8ac8-a7e7a8df06d0.PNG)
![Menu](https://user-images.githubusercontent.com/72570229/96374443-82a8de80-1190-11eb-984b-0244a85c9887.PNG)

**Comment by johan7004** [comment]
can i work on this ?

**Comment by janesteffi** [comment]
yes

**Comment by janesteffi** [comment]
any updates ? its been a while......

**Comment by johan7004** [comment]
almost done will close the issue shortly.. regret for the late update

**Comment by johan7004** [comment]
hey, i have sent an pull request can you merge?

**Comment by johan7004** [comment]
the issue is not closed yet is there anything i am missing ?

**Comment by janesteffi** [comment]
No, technically you solved the issue i was just waiting for more contributors. I am planning to wait for few more days to see if there is any response, to be honest the response is not as i expected.

**Comment by johan7004** [comment]
is there any automation analysis done by you on the code which I sent ?

**Comment by janesteffi** [comment]
yes I have recently performed automated analysis on the site, which came out as expected. Thank you for your contribution. feel free to work on my other repositories.

**Comment by johan7004** [comment]
thank you for the feedback
